lib_xua/src/core/endpoint0/descriptor_defs.h

```diff
@@ -1,20 +1,20 @@
-// Copyright 2015-2023 XMOS LIMITED.
+// Copyright 2015-2024 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
 
-#ifndef __DESCRIPTOR_DEFS_H__
-#define __DESCRIPTOR_DEFS_H__
+#ifndef _DESCRIPTOR_DEFS_H_
+#define _DESCRIPTOR_DEFS_H_
 
 /*
     Include xua.h to pick up the #defines of NUM_USB_CHAN_IN and NUM_USB_CHAN_OUT.
  */
 #include "xua.h"
 
 #if (NUM_USB_CHAN_IN > 0) && (NUM_USB_CHAN_OUT > 0)
-#define AUDIO_INTERFACE_COUNT 3
+#define AUDIO_INTERFACE_COUNT (3)
 #elif (NUM_USB_CHAN_IN > 0) || (NUM_USB_CHAN_OUT > 0)
-#define AUDIO_INTERFACE_COUNT 2
+#define AUDIO_INTERFACE_COUNT (2)
 #else
-#define AUDIO_INTERFACE_COUNT 1
+#define AUDIO_INTERFACE_COUNT (0)
```

Comment:
What was the meaning of the 1 before? Do we never have a case where it is 1 now?